### DISCUSSION TITLE
Execute custom action while uninstalling HotFix

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#6877] [Questions] [answered]
I have included a custom action in Hotfix with condition "1" so that my custom action will execute in all phases like install, repair and uninstall.
However my custom action is executing only during install and repair phase. It is not executing in uninstall phase.
i have seen logs files. My custom action is logged in install and repair log files. But not logged in uninstall log file(not even showing skipped).
How to execute custom action while uninstalling HotFix.

Below is my custom action:
```
<InstallExecuteSequence>
 <Custom Action="DeleteCATest"  Before="InstallFiles">1</Custom>
</InstallExecuteSequence>
  <CustomActionRef Id="DeleteCATest"/>
```

**Comment by robmen** [comment, marked as answer]
I'm guessing by "Hotfix" you mean MSI Patch (aka: MSP). Executing [custom actions during patch uninstall has a dedicated documentation page](https://docs.microsoft.com/en-us/windows/win32/msi/patch-uninstall-custom-actions).
